bfunwind/src/dwarf4.cpp

```diff
@@ -183,22 +202,1287 @@ class cfi_table_row
 // Unwind Helpers
 // -----------------------------------------------------------------------------
 
+// TODO: In the future, the stack should be implemented using a custom C++
+//       class that guards [] to prevent the stack variable from being
+//       accessed out of bounds
+
+static uint64_t
+private_parse_expression(char *p,
+                         uint64_t initialStackValue,
+                         register_state *state)
+{
+    uint64_t i = 0;
+    uint64_t stack[100];
```

Comment:
The DWARF spec does not define a stack size. You have to pick what you want to support. We just do the same thing Apple is doing